Twig/DoctrineExtension.php

```diff
@@ -314,6 +315,33 @@ function ($matches) use ($parameters, &$i) {
     }
 
     /**
+     * Highlights the syntax of the given SQL statement.
+     *
+     * @param  string $sql
+     * @return string
+     */
+    public function highlightSqlSyntax($sql)
+    {
+        $highlighter = new \SqlFormatter();
```

Comment:
no need to create an instance. SqlFormatter is a procedural API wrapped in a class anyway, and you are still changing the global state even if you access static params through the instance rather than the class name